pkg/btf/btf_test.go

```diff
@@ -44,6 +46,25 @@ func setupfiles() func(*testing.T, string, ...string) {
 	}
 }
 
+func listBtfFiles() ([]string, error) {
+	kernelVersion, err := forceConfiguredKernelVersion()
+	if err != nil {
+		return nil, err
+	}
+
+	_, testFname, _, _ := runtime.Caller(0)
+	testdataPath := filepath.Join(filepath.Dir(testFname), "..", "..", "testdata")
+
+	btfFiles := []string{
+		defaults.DefaultBTFFile,
+		path.Join(defaults.DefaultTetragonLib, "btf"),
+		path.Join(defaults.DefaultTetragonLib, "metadata", "vmlinux-"+kernelVersion),
+		filepath.Join(testdataPath, "btf", "vmlinux-5.4.104+"),
```

Comment:
so there will be some directory structure based on test name? could you describe that in detail?

Reply:
This is absolute paths to the files. Should I add some documentation above the code to describe it more ? 
Basically I simply took the code from elsewhere with no changes

Reply:
`path.Join(defaults.DefaultTetragonLib, "btf"),` and `path.Join(defaults.DefaultTetragonLib, "metadata", "vmlinux-"+kernelVersion),`

Come from `observerFindBTF`

https://github.com/cilium/tetragon/blob/84212ddf3a8cb0fe5df2e77fe497f11e859996a5/pkg/btf/btf.go#L66
https://github.com/cilium/tetragon/blob/84212ddf3a8cb0fe5df2e77fe497f11e859996a5/pkg/btf/btf.go#L59

And `filepath.Join(testdataPath, "btf", "vmlinux-5.4.104+"),` 
From `TestSpec`
https://github.com/cilium/tetragon/blob/84212ddf3a8cb0fe5df2e77fe497f11e859996a5/pkg/btf/validation_test.go#L66